assemblies/installing-odh-v2.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: ASSEMBLY
+//pv2hash: 
+
+ifdef::context[:parent-context: {context}]
+
+:context: installv2
+
+[id="installing-odh-v2_{context}"]
+= Installing Open Data Hub (v2)
+
+[role='_abstract']
+This section describes how to install the Open Data Hub Operator v2.X on your OpenShift Container Platform from the OpenShift web console. 
```

Comment:
try to avoid "This section describes" 
probably don't need 2.x, just use "version 2"

replace with something like 
"You can install the Open Data Hub Operator version 2 on OpenShift Container Platform from the OpenShift console. "